src/inverseRobot.js

```diff
@@ -7,7 +7,30 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+  const keys = Object.values(robot);
+  const values = Object.keys(robot);
+  let count = 0;
+
+  for (const key of keys) {
+    for (let i = 0; i < keys.length; i++) {
+      if (key === keys[i]) {
+        count++;
+      }
+    }
+
+    if (count > 1) {
+      return null;
+    }
+
+    count = 0;
+  }
+
+  for (let i = 0; i < keys.length; i++) {
+    result[keys[i]] = values[i];
+  }
```

Comment:
we need only one condition inside the loop to check current key
```
if(result[robot[key]) {
  return null
}
```

and if this condition is falsy we just update our object
`result[robot[key]] = key;`